Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,122 @@
 - Engineers at Octopus seek out to work on my teams to learn from me.
 - After reviewing a code contribution from an engineer I saw could be improved, I worked directly with that engineer and their manager or team leader to uplift their skills in the identified growth area.
 - I helped a team make a course correction based on new information or ideas.
 - I provided appropriate technical freedom, and framing, that allowed the Lead Software Engineers on my teams to thrive independently.
+- I have elevated my focus significantly because the teams I support have been uplifted and empowered to the point they can deliver their mission with me as a sounding board.
 
 </details>
 
 ## 🧭 Culture and Leadership
 
-- I create an environment where others feel safe, by modeling vulnerability, thinking in the open, being wrong, and accepting corrections.
-- I generate energy within the teams I work with, motivating them to achieve great outcomes.
+- I am a proponent of [operational excellence](https://en.wikipedia.org/wiki/Operational_excellence) and [learning from incidents](https://www.learningfromincidents.io/about).
+- I create an environment where others feel safe to take risks and make mistakes, by modeling vulnerability, thinking in the open, being wrong sometimes, and accepting corrections.
+- I generate energy in the teams I work with, motivating them to achieve great outcomes.
 - I facilitate effective and timely decision making, especially when the decision affects multiple teams.
 - I build solutions that improve the productivity and/or quality of work of many people/teams.
+- I recognise patterns of organisational dysfunction, by observing data points, amplifying these signals, and advocating effectively for improvement.
+- I am a contributor to or member of the leadership teams in my group.
 - I set a positive example throughout the company for quality, accountability, and responsibility.
-- I actively share knowledge relevant to product delivery teams.
 - I help define hiring standards and practices.
 
 <details>
 <summary>Examples</summary>
 
-- I drove an entire multiteam program from inception through to shipping code, without regular technical oversight.
+- During an incident postmortem, I recognised a valuable lesson and found a way to disseminate the lesson throughout R&D.
+- I drove an entire multiteam program from inception through to shipping code, without regular oversight.
 - I collaborated with other senior leaders to ensure I was aware of all major initiatives at Octopus and could account for them in my own initiatives.
-- I identified a significant problem/opportunity and created a lasting best-fit solution that aligned with our strategy.
+- The teams I work with are consistently winning. They chose ambitious targets, managed scope and expectations effectively, and delivered customer value frequently with customer adoption trends going in the right direction.
```

Comment:
nit:

```
- The teams I work with are consistently winning. They chose ambitious targets, managed scope and expectations effectively, and delivered customer value frequently with customer adoption trends going in the right direction.
```

"consistently winning" could imply there isn't room for making a mistake or learning from a failure.

Suggest a change to:
```
- The teams I work with chose ambitious targets, managed scope and expectations effectively, and delivered customer value frequently with customer adoption trends going in the right direction.
```

Or if this relates to:
```
- I generate energy in the teams I work with, motivating them to achieve great outcomes.
```

Maybe we want:
```
- The teams I work with feel like they are winning. They chose ambitious targets, managed scope and expectations effectively, and delivered customer value frequently with customer adoption trends going in the right direction.
```

Reply:
>"consistently winning" could imply there isn't room for making a mistake or learning from a failure.

Good pick up. Thanks.

I do want to retain an aspect of teams having that winning feeling. I've adopted your last suggestion.